src/javaharness/java/arcs/api/StorageProxy.java

```diff
@@ -8,17 +8,16 @@
   public final String id;
   public final String name;
   public final Type type;
-  protected boolean listenerAttached;
-  protected Map<Handle, NativeParticle> observers;
+  protected boolean listenerAttached = false;
+  protected boolean keepSynced = false;
+  protected Map<Handle, NativeParticle> observers = new HashMap<Handle, NativeParticle>();
```

Comment:
You can reduce the chickenchicken here with type inference: `Map<Handle, NativeParticle> observers = new HashMap<>();`

Reply:
done, thanks!